### PR TITLE
Add new tests and helpers, update genesis files

### PR DESCRIPTION
**Pull request by VladimirWork**

### CODE REVIEW

File: system/buildernet_genesis [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{"reqSignature":{},"txn":{"data":{"data":{"alias":"FoundationBuilder","blskey":"3gmhmqpPLqznZF3g3niodaHjbpsB6TEeE9SpgXgBnZJLmXgeRzJqTLajVwbhxrkomJFTFU4ohDC4ZRXKbUPCQywJuPAQnst8XBtCFredMECn4Z3goi1mNt5QVRdU8Ue2xMSkdLpsQMjCsNwYUsBguwXYUQnDXQXnHqRkK9qrivucQ5Z","blskey_pop":"RHWacPhUNc9JWsGNdmWYHrAvvhsow399x3ttNKKLDpz9GkxxnTKxtiZqarkx4uP5ByTwF4kM8nZddFKWuzoKizVLttALQ2Sc2BNJfRzzUZMNeQSnESkKZ7U5vE2NhUDff6pjANczrrDAXd12AjSG61QADWdg8CVciZFYtEGmKepwzP","client_ip":"35.161.146.16","client_port":"9702","node_ip":"50.112.53.5","node_port":"9701","services":["VALIDATOR"]},"dest":"GVvdyd7Y6hsBEy5yDDHjqkXgH8zW34K74RsxUiUCZDCE"},"metadata":{"from":"V5qJo72nMeF7x3ci8Zv2WP"},"type":"0"},"txnMetadata":{"seqNo":1,"txnId":"fe991cd590fff10f596bb6fe2362229de47d49dd50748e38b96f368152be29c7"},"ver":"1"}
```
Comment (ashcherbakov) [inline system/buildernet_genesis]: Isn't it the same files as in `sovrin` repo? Why don't take files from there?

File: system/draft/TestProductionSuite.py [hunk 2]
```diff
@@ -33,7 +33,8 @@ class TestProductionSuite:
         )
 
         # put both genesis files into extra nodes
-        assert all([c.exec_run(['mkdir', GENESIS_PATH], user='indy').exit_code == 0 for c in extra_containers])
+        print([c.exec_run(['mkdir', GENESIS_PATH], user='indy') for c in extra_containers])
```
Comment (ashcherbakov) [inline system/draft/TestProductionSuite.py]: Why assert has been replaced by print?

File: system/draft/test_upgrade.py [hunk 3]
```diff
@@ -112,18 +126,18 @@
...
-    await asyncio.sleep(7*5*60)
+    await asyncio.sleep(25*5*60)
     # await asyncio.sleep(180)
 
-    docker_7_hosts = [
-        testinfra.get_host('docker://node' + str(i)) for i in range(1, 8)
-    ]
-    # aws_25_hosts = [
-    #     testinfra.get_host('ssh://persistent_node'+str(i), ssh_config='/home/indy/.ssh/config') for i in range(1, 26)
+    # docker_7_hosts = [
```
Comment (ashcherbakov) [inline system/draft/test_upgrade.py]: Does it make sense to have 2 tests, one for 7 nodes Docker pool and one for 25 nodes AWS pool

File: system/draft/test_misc.py [hunk 4]
```diff
@@ -96,28 +90,67 @@
     print(res)
 
 
+# TODO parametrize this to read as trustee and as default user
 @pytest.mark.asyncio
 async def test_misc_state_proof(
```
Comment (ashcherbakov) [inline system/draft/test_misc.py]: What exactly does this test checks? Please consider writing a short description for every test.
For example:
- add X NYM txns
- read written NYM txns
- make sure that ....

File: system/draft/test_misc.py [hunk 5]
```diff
@@ -1649,10 +1793,47 @@ def test_misc_hypothesis_sync(docker_setup_and_teardown, pool_handler, wallet_ha
...
 
 
 @pytest.mark.asyncio
-async def test_misc_draft(
-        docker_setup_and_teardown, pool_handler, wallet_handler, get_default_trustee
+async def test_misc_get_auth_rule():
+    pool_handle, _ = await pool_helper(path_to_genesis='/home/indy/indy-test-automation/system/buildernet_genesis')
+    req = await ledger.build_get_auth_rule_request(None, None, None, None, None, None)
+    res = json.loads(await ledger.submit_request(pool_handle, req))
+    print('\n', res)
```
Comment (ashcherbakov) [inline system/draft/test_misc.py]: What is assertion in this test?

File: system/draft/test_misc.py [hunk 6]
```diff
@@ -96,28 +90,67 @@
     print(res)
 
 
+# TODO parametrize this to read as trustee and as default user
 @pytest.mark.asyncio
```
Comment (ashcherbakov) [inline system/draft/test_misc.py]: Please make sure that all tests are run as part of nightly builds.

File: system/draft/test_upgrade.py (hunk 3 above)
Comment (VladimirWork) [reply system/draft/test_upgrade.py]: Yes, I will split it.

File: system/draft/TestProductionSuite.py (hunk 2 above)
Comment (VladimirWork) [reply system/draft/TestProductionSuite.py]: We have post install scripts in sovrin that create this folder but does not have them in indy-node so this assert will fail against sovrin setup.

File: system/draft/test_misc.py (hunk 6 above)
Comment (VladimirWork) [reply system/draft/test_misc.py]: It will be done in the next PR.

File: system/buildernet_genesis (hunk 1 above)
Comment (VladimirWork) [reply system/buildernet_genesis]: Helpers will be implemented for this actions.

File: system/draft/test_misc.py (hunk 4 above)
Comment (VladimirWork) [reply system/draft/test_misc.py]: It will be in the next PR. Also it will be a separate test.

File: system/draft/test_misc.py (hunk 5 above)
Comment (VladimirWork) [reply system/draft/test_misc.py]: It's just a draft test, removed.

**Pull request merged**